SofaKernel/modules/SofaHelper/CMakeLists.txt

```diff
@@ -188,10 +185,6 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL Darwin)
 elseif(CMAKE_SYSTEM_NAME STREQUAL Windows)
     list(APPEND SOURCE_FILES  ${SRC_ROOT}/system/FileMonitor_windows.cpp)
 endif()
-if(Boost_FILESYSTEM_FOUND)
-    list(APPEND HEADER_FILES ${SRC_ROOT}/system/FileRepository.h)
-    list(APPEND SOURCE_FILES ${SRC_ROOT}/system/FileRepository.cpp)
-endif()
```

Comment:
We still need FileRepository !

Reply:
look at the SOURCE_FILES closely...
(hint, yes it was given twice 🤥)

Reply:
Nevermind :roll_eyes: